server.js

```diff
@@ -3,5 +3,14 @@
 const {DashServer} = require('./server/lib/server');
 const {GitHub} = require('./server/lib/github');
 
+try {
+  const secrets = require('./secrets.json');
+  if (secrets['GITHUB_CLIENT_SECRET']) {
+    process.env.GITHUB_CLIENT_SECRET = secrets['GITHUB_CLIENT_SECRET'];
+  }
+} catch (err) {
+  // No secrets defined.
+}
```

Comment:
Probably should remove the try catch or throw the error? The app can't really run without these secrets, so best to cause an exception.